sysbuild/Kconfig.matter

```diff
@@ -39,4 +39,7 @@ config MATTER_OTA_IMAGE_FILE_NAME
 	help
 	  Provides the file name of the generated Matter OTA image in the build directory.
 
+config SUIT_ENVELOPE_ROOT_TEMPLATE
+	default "../common/suit_templates/root_with_binary_nordic_top.yaml.jinja2"
```

Comment:
using a relative path from within a specific sample can make sense as you know the location of the sample itself.

But making the default relative to a sample's path when you don't know the sample being built looks wrong to me, and something which is bound to fail at some point.

Is is not possible to have the location defaulting to a path within a specific Zephyr module, such as ZEPHYR_NRF_MODULE_DIR ?

Reply:
I checked that and indeed using the `ZEPHYR_NRF_MODULE_DIR` variable works, but if I use it within the sample directory, it points to the `<workdir>/ncs/nrf/samples/matter/template/` instead of `<workdir>/ncs/nrf`, but while using it within the `sysbuild/Kconfig.matter` file it works in the proper way and points to the `<workdir>/ncs/nrf` I have no idea why...